src/lib/app-fns/tx/common/sending.tsx

```diff
@@ -14,7 +14,7 @@ export const sendingTx = (fee: StdFee) => {
       receipts: [
         {
           title: "Estimated Tx Fee",
-          value: formatStdFee(fee),
```

Comment:
actually other files use tx fee from tx hash - should we do the same for this file?

```
const txFee = txInfo.events.find((e) => e.type === "tx")?.attributes[0]
        .value;
```

Reply:
maybe `coinsFromStr` can help with parsing